docs/cookbook.rst

```diff
@@ -198,7 +208,9 @@ Sometimes you only need logging when things go wrong.
 to only log the important things when things go right.
 Everything inside of ``Silently`` is prevented from logging.
 
-Example: The following Action::
+Example: The following Action
```

Comment:
... this one should have the `:` here, or this line should be reworded a bit.

Reply:
fixed